README.md

```diff
@@ -129,6 +129,40 @@ Examples:
 - `udp://localhost:4000`
 - `udp://localhost:4000?local_port=4001`
 
+**The Connection Refused error** If configured to do so, UDP streams will report
+a connection refused error if there are no processes listening on the configured
+remote peer. This is controlled by the `ignore_connrefused` parameter which has
+to be set to 0 or 1. For backward compatibility reasons, the default behavior of
+UDP streams with respect to this option is complex, see below for details.
+
+**Connected UDP sockets** If configured to do so, UDP streams are _connected_,
+that is will only accept packets from the configured remote host. If unconnected,
+they will receive from any host (but still send to the configured host). This
+is controlled by the `connected` parameter which has to be set to 0 or 1. For
+backward compatibility reasons, the default behavior of UDP streams with
+respect to this option is complex, see below for details.
+
+**Default `connected` and `ignore_connrefused` parameters**
+
+For backward compatibility reasons, the default values for `connected` and
+`ignore_connrefused` depends on whether the UDP stream was created with or without
+a local port. The behavior described below is deprecated. In the future, the default
+will be set to `connected=1` and `ignore_connrefused=1`. Warnings are currently issued
+when the current defaults are used. Explicitely set these parameters to shut the
+warnings and ensure your code will continue working as-is when the defaults change.
+
+- `ignore_connrefused` may be set to zero (to report connrefused) only on connected
+  sockets. Attempting to set `ignore_connrefused=0&connected=0` will throw in `openURI`
```

Comment:
Is there a reason for not allowing to ignore ECONNREFUSED on unconnected sockets?

Reply:
According to my test, recvfrom does not report ECONNREFUSED on non-connected sockets.

Reply:
Didn't know that. :+1: